doc/usecases/gamification/map_collaboration.md

```diff
@@ -0,0 +1,32 @@
+# Use Case: Map Collaboration
+
+## Summary
+
+- **Scope:** Map Collaboration
+- **Level:** User Goal
+- **Actors:** App User
```

Comment:
```suggestion
- **Actors:** Map Creator, Collaborator
```
Adjust the rest of the use case to use these terms.

Reply:
Good suggestion!